src/index.js

```diff
@@ -321,7 +321,7 @@ function localizeDeclaration(declaration, context) {
   const isAnimation = /animation$/i.test(declaration.prop);
 
   if (isAnimation) {
-    const validIdent = /^-?[_a-z][_a-z0-9-]*$/i;
+    const validIdent = /^-?(?:\\[~!@$%^&*()+=,./';:"?><[]\\{}|`#]|[_a-z])(?:\\[~!@$%^&*()+=,./';:"?><[]\\{}|`#]|[_a-z0-9-])*$/i;
```

Comment:
Can you show where you get this regexp :smile: Ident token from spec https://www.w3.org/TR/css-syntax-3/#typedef-ident-token

Reply:
That spec seems to only explain escaping Unicode points (ie `\00002b`)

They have a diagram labeled `escape` which seems to indicate the character following the escape can be any character except newline - the spec explicitly says that the diagrams are not 100% accurate though, so I'm not sure how much stock I should put in that.

I got this list from the following post:
https://mathiasbynens.be/notes/css-escapes

Cross referencing some SO posts seemed to confirm this list. 

Would it be better to have a more relaxed regex around escapes?

Reply:
I am fine with it, just want to verify

Reply:
Is there anything else you would like me to address before getting these changes in?

Reply:
@alexander-akait I hope I'm not bothering. Is this MR ready to go?

Reply:
Would limiting the scope of these changes to the special character `@` speed this MR along? That is the only character currently affecting  our projects.

Reply:
Sorry for delay, yes, I think let's do it to avoid big perf regression